src/utilities/buildClientSchema.js

```diff
@@ -381,13 +381,14 @@ export function buildClientSchema(
   }
 
   function buildDirective(directiveIntrospection) {
-    const directiveIntrospectionStr = inspect(directiveIntrospection);
```

Comment:
@spawnia `inspect` is not a zero-cost some we shouldn't call it unless we throw an error.
I think in this case is better to duplicate code.

Reply:
Good point, makes sense.